stable/artifactory-ha/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # JFrog Artifactory-ha Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [4.1.1] - Oct 06, 2020
```

Comment:
This is a breaking change. Please say that in the changelog. Also, this justifies a minor version bump IMO

Reply:
will update

Reply:
Done